peerscout/utils/html.py

```diff
@@ -0,0 +1,25 @@
+from html.parser import HTMLParser
+
+
+class MLStripper(HTMLParser):
+    def __init__(self):
+        super().__init__()
+        self.reset()
+        self.strict = False
+        self.convert_charrefs = True
+        self.fed = []
+
+    def handle_data(self, data):
+        self.fed.append(data)
+
+    def get_data(self):
+        return ''.join(self.fed)
+
+    def error(self, message):
```

Comment:
i can't really work out what this is for

Reply:
Do you mean this particular error method? It's a good question. I have to admit I had copied and used it before without reviewing it properly. There doesn't seem to be any reference in the [docs](https://docs.python.org/3/library/html.parser.html). I shall remove it.

Reply:
Actually it is kind of required because the [error method in ParserBase](https://github.com/python/cpython/blob/v3.8.2/Lib/_markupbase.py#L32-L34) is semi-abstract. `pylint` detects that (even though it's not annotated as such). Because ParserBase doesn't actually use `@abstractmethod` it is working. But if an error occurred, it would have raised a `NotImplementedError` error instead of the actual parsing error. Seems like an omission in `HTMLParser` to implement the method (or `ParserBase` could provide a sensible default behaviour). I will add a comment.